.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: scala
+
+sudo: required
+
+scala:
+   - 2.11.8
+
+jdk:
+   - oraclejdk8
+
+services:
+  - docker
+
+before_install:
+  - docker-compose up -d
+
+script:
+  - sbt clean coverage test coverageReport coverageAggregate
```

Comment:
We are doing nothing with the coverage. I propose to remove all coverage things for now and add them later again in a second PR (where we can also configure codacy properly)